mitodl/mit-open
# add learning materials tab

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4620

### Description (What does it do?)
This PR adds a "Learning Materials" tab to the search page and channel pages

### Screenshots (if ap
<img width="1730" alt="Screenshot 2024-06-20 at 4 47 03 PM" src="https://github.com/mitodl/mit-open/assets/9010790/aaf78589-946f-4f97-8731-1471c4cef35d">
propriate):
<img width="1731" alt="Screenshot 2024-06-20 at 4 35 03 PM" src="https://github.com/mitodl/mit-open/assets/9010790/7f6aaa01-fc4f-4cea-a7d4-5385d559cec0">
<img width="1730" alt="Screenshot 2024-06-20 at 4 35 18 PM" src="https://github.com/mitodl/mit-open/assets/9010790/cda4b1a5-ee4e-4c94-8b63-1befd4aa9e85">
<img width="1732" alt="Screenshot 2024-06-20 at 4 35 37 PM" src="https://github.com/mitodl/mit-open/assets/9010790/57a85a12-d926-4076-8f3b-828215fca44b">

### How can this be tested?
On this branch:
1. View the search page locally. 
2. You should see 4 tabs: "All", "Courses", "Programs", "Learning Materials".
    - Check that each tab shows the appropriate things
    - "Learning Materials" should show everything except courses and programs
3. Check that tab counts do not change when switching tabs
